### DISCUSSION TITLE
Attempting to login, receive "email address used" error, go to log in with email, receive "no account with that email exists" error

### DISCUSSION THREAD

**Discussion by robswn** [vercel/community#7216] [Account Issue]
### Question

Hi! 
I am trying to close a previous account used for Vercel (GitHub account under `rob+vercel@lovelight.com.au` ) and create a new account associated with this GitHub account, under `rob@lovelight.com.au`.

However, when I try and login with GitHub it tells me an account using that email already exists and I have to log in using email. When I attempt to log in using email, it tells me no account under that name exists.

I can't seem to resolve this loop of errors! There's no sensitive or precious data associated with the account so happy to just have it scrubbed entirely. Thanks for your help!

**Comment by robswn** [comment]
Additionally, when I try to get my admin to invite me, I get the following error:

![image](https://github.com/vercel/community/assets/86272367/9d084922-c6de-4122-8b73-4e7879ddbc7b)

**Reply by pawlean** [reply]
Hi @robswanLL! 

Thanks for your patience 🙏🏼 Are you still having this issue?

[a comment by an automated account is not shown]
